app/base/static/assets/lightbox2-2.11.3/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,4 @@
+> Pull Requests are welcome. But v2 of Lightbox is in Maintenance Mode. 
```

Comment:
Light box can simply be installed by downloading the library which has js, some images and css files 
This looks like you cloned a repository of it, this is not right.
Please get the library specific installer

Reply:
The download link on their page takes to Github. So I download it on Github, extracted the files, and copied the `dist` folder to `static/assets` since the `dist` folder contains the needed files.